### ISSUE TITLE
The users interface

### ISSUE THREAD

**Issue by Ngo-Bassom-Anne-Rosa** [PacomeKFP/snappy#4]
![Screenshot from 2024-12-18 01-13-58](https://github.com/user-attachments/assets/d4b8b373-0926-4090-bdfe-466d28b87c0f)
This is the users interface, for me to do it as simple as possible we can just do an array, with columns like username, email, adduser to my list , discuss with the user . This array, will permit us to have a fluid list of users, and give the capacity to the owner of the session to easily start writing to somebody who is not from his contact list but from the list of the app, and then he can add the user directly to his contact list and ecrease his list.

**Comment by PacomeKFP** [comment]
Pay attention, it seem like you are confusing types of users, 

This UIs a made for Orgs like Adrien and other services, they don't need to send messages to this dashboard, the only use it to manage the a chat app, (adding/removing users, is the main feature, they can also come on this app to have some importants tokens to create a new chat app and consume our api)

**Comment by PacomeKFP** [comment]
Yes, it hava ti be a table, where colums will represents user attributes (username, image....)
